.teamsnap/service.yml

```diff
@@ -1,15 +1,18 @@
 ---
-classification: 
+classification: lib
```

Comment:
tool makes the most sense to me. it is more than a lib but less than an internal app

Reply:
It is an internal tool that I created from a 10% project. Helps the marketers collaborate on responsive email. 

https://ts-mar-email.now.sh/

Reply:
Thanks @l3aconator where is that running? Is that now.sh like a netlify?

Reply:
It used to be Now (folks who run next.js), but is technically, https://vercel.com/. I built this like 2 years ago so def needs some love on the hosting/updating front. I can migrate over to netlify, just would have to take advantage of the cloud functions, that's why it is on Vercel as that was free. Either way, they are lambdas, requests are super low so shouldn't be a big issue.

@ryanwi